src/codemodder/codemods/import_modifier_codemod.py

```diff
@@ -52,9 +52,6 @@ def mapping(self) -> Mapping[str, str]:
         pass
 
     def transform_module_impl(self, tree: cst.Module) -> cst.Module:
-        if not self.node_is_selected(tree):
```

Comment:
this would prevent any tool results from moving forward. doesn't seem like it's currently in use via testing